src/utils.ts

```diff
@@ -252,3 +252,20 @@ export function spanLogic(
     rightLastVisible,
   };
 }
+
+export function getMaxDepth(rootNodes:Array<any>){
```

Comment:
why not fixing the argument to be `rootNodes: ILabelNode[]`?

Reply:
done